test/tests/vectorpostprocessors/element_variables_difference_max/element_variables_difference_max_TT.i

```diff
@@ -0,0 +1,89 @@
+[Mesh]
+  type = GeneratedMesh
+  dim = 3
+  nx = 10
+  ny = 10
+  nz = 10
+[]
+
+[Variables]
+  [./u]
+    order = FIRST
+    family = LAGRANGE
+  [../]
+
+  [./v]
+    order = FIRST
+    family = LAGRANGE
+  [../]
+[]
+
+[Functions]
+  [./forcing_v]
+    type = ParsedFunction
+    value = 'sin(pi * x) * sin(pi * y) * sin(pi * z)'
+  [../]
+[]
+
+[Kernels]
+  [./diffusion_u]
+    type = Diffusion
+    variable = u
+  [../]
+  [./time_u]
+    type = TimeDerivative
+    variable = u
+  [../]
+
+  [./diffusion_v]
+    type = Diffusion
+    variable = v
+  [../]
+  [./forcing_v]
+    type = UserForcingFunction
+    variable = v
+    function = forcing_v
+  [../]
+  [./time_v]
+    type = TimeDerivative
+    variable = v
+  [../]
+[]
+
+[BCs]
+  [./bottom]
+    type = DirichletBC
+    variable = 'u v'
+    boundary = 'bottom'
+    value = 1
+  [../]
+
+  [./top]
+    type = DirichletBC
+    variable = 'u v'
+    boundary = 'top'
+    value = 0
+  [../]
+[]
+
+[VectorPostprocessors]
+  [./differenceTT]
+    type = ElementVariablesDifferenceMax
+    compare_a = u
+    compare_b = v
+    furthest_from_zero = true
+    preserve_sign = true
```

Comment:
Look at [moose/test/tests/mesh/high_order_elems/tests for example](https://github.com/idaholab/moose/blob/devel/test/tests/mesh/high_order_elems/tests) for example